x-pack/plugins/security_solution/server/lib/detection_engine/signals/source_fields_merging/strategies/merge_no_fields.ts

```diff
@@ -10,6 +10,7 @@ import { MergeStrategyFunction } from '../types';
 /**
  * Does nothing and does not merge source with fields
  * @param doc The doc to return and do nothing
+ * @param ignoreFields We do nothing with this value and ignore it if set
  * @returns The doc as a no operation and do nothing
  */
 export const mergeNoFields: MergeStrategyFunction = ({ doc }) => doc;
```

Comment:
nit: I know it's not used, but since you added the JSDoc, and it's not optional in `MergeStrategyFunction`.

```suggestion
export const mergeNoFields: MergeStrategyFunction = ({ doc, ignoreFields }) => doc;

```